backend/crawlers/ranking.js

```diff
@@ -9,10 +9,13 @@ const {
   wait,
   dbQuery,
   dbParamQuery,
+  getCrawlerConfigByName,
 } = require('../lib/utils');
 const backendConfig = require('../backend.config');
 
 const crawlerName = 'ranking';
+const crawlerConfig = getCrawlerConfigByName(backendConfig, crawlerName);
```

Comment:
`config` is already in place. Please, use it

Reply:
Done